ledger/src/ui/src/pin.h

```diff
@@ -27,8 +27,79 @@
 
 #include <stdbool.h>
 
-#define PIN_LENGTH 8
+#define MAX_PIN_LENGTH 8
+#define MIN_PIN_LENGTH 4
```

Comment:
Are we using `MIN` and `MAX`? I thought we were only accepting 8 character pins.

Reply:
We accept pins with less than 8 chars on debug mode, but you are correct, we never use the `MIN_PIN_LENGTH` macro. I'll change that to `PIN_LENGTH`.